lib/src/models/usage.dart

```diff
@@ -41,27 +41,34 @@ class PlanUsageBreakdown {
 
 /// Contains main information about usage.
 class Usage {
+
+  /// End date of a billing cycle.
+  DateTime? billingCycleEnd;
+
+  /// Start date of a billing cycle.
+  DateTime? billingCycleStart;
+
+  /// Indicator if the Explorer has access to a 5 min free call.
+  bool? hasFreeCall;
+
   /// Name of the plan.
   String? planName;
 
-  /// Total available minutes in the plan.
-  int? totalMinutes;
+  /// Indicates if the plan is unlimited.
+  bool? planUnlimited;
```

Comment:
I think we will be getting rid of unlimited plans. If we don't, maybe we can eventually eliminate this and instead set `totalPrimaryMinutes` to `-1` (a pattern for "unlimited" used elsewhere in Platform).